src/Microsoft.TestPlatform.CrossPlatEngine/Resources/xlf/Resources.pt-BR.xlf

```diff
@@ -155,7 +155,8 @@
       <trans-unit id="TestHostExitedWithError">
         <source>Testhost process exited with error: {0}. Please check the diagnostic logs for more information.</source>
         <target state="translated">O processo do testhost foi encerrado com o erro: {0}. Verifique os logs de diagnóstico para obter mais informações.</target>
-        <note></note>
+        <note>
```

Comment:
@v-zbsail Why does this show up as a change ? We should instead change this to <note/> instead. ?

Reply:
Hi @singhsarab, this behavior is by design done in our building tool (adding new line between <note> tags). If it causes problems on your side, we can create post-process on our side to remove these.

Reply:
Yeah, please do, the xlf tool creates those for us, and we don't want to keep going back and forth on this.

Reply:
I'll keep a note on this and make sure to have those correct next time! Thanks